src/meta_api.cpp

```diff
@@ -113,6 +113,7 @@ C_DLLEXPORT int Meta_Detach(PLUG_LOADTIME now [[maybe_unused]],
     fwdMngr->getDefaultForward(def::PluginEnd)->execFunc(nullptr);
 
     gSPGlobal->getPluginManagerCore()->clearPlugins();
+    gSPGlobal->getTimerManagerCore()->clearTimers();
```

Comment:
`Meta_Detach` called after meta plugin unload but we need clear timers after changelevel.
We load spmod plugins after any restart, changelevel and register new Timers.
Etc for commands, natives, forwards registered from spmod plugin.